src/DynamoPackagesUI/Utilities/PublishCommands.cs

```diff
@@ -23,7 +23,7 @@ internal class PublishCefHelper : CefHelper
 
         private PackageManagerViewModel packageMgrViewModel { get; set; }
 
-        public PublishCefHelper(DynamoViewModel dynamoViewModel, PackageLoader model, PackageManagerViewModel pkgManagerViewModel) : base(dynamoViewModel, model, pkgManagerViewModel)
+        public PublishCommands(DynamoViewModel dynamoViewModel, PackageLoader model, PackageManagerViewModel pkgManagerViewModel) : base(dynamoViewModel, model, pkgManagerViewModel)
```

Comment:
We refer DynamoViewModel at multiple places while implementing Explore Packages functionality install, uninstall packages etc... its the same kind of code as DynamoCoreWpf/PackageManager